.travis.yml

```diff
@@ -1,18 +1,14 @@
 language: python
+dist: xenial
 python:
-- 2.7
-- 3.4
-- 3.5
-- 3.6
-- 3.7-dev
-- pypy
```

Comment:
I understand that we're dropping support for Python v2 and older versions 3, but are we also dropping pypy?